ramda/ramda
# chore: update vulnerable dependencies

### Description
- updates packages with security issues. 

Fixes #3419  

### Type of change
- [ ] Chore(non-breaking change which adds functionality)


### How Has This Been Tested?
- Update the dependencies via `npm install`
- Run all tests to ensure there's no breakage
